### PR TITLE
refactor: update nmrium to use react-analysis-ui

### PR DESCRIPTION
**Pull request by Sebastien-Ahkrin**

### CODE REVIEW

**Comment by Sebastien-Ahkrin** [comment]
* Error while you add a new molecule on the bottom right panel

**Comment by lpatiny** [comment]
There is a pixel shift when you select a tool that changes the menu bar that should be fixed

**Comment by targos** [comment]
- Fix the sidebar so it has the same size as the one on dev.nmrium.org. (in pixels, not percent)
- increase the size of the toolbar icons

File: src/component/NMRium.tsx [hunk 1]
```diff
@@ -11,12 +12,10 @@ import {
   useRef,
   memo,
   Reducer,
-  CSSProperties,
   ReactElement,
   ReactNode,
 } from 'react';
 import { ErrorBoundary } from 'react-error-boundary';
-import SplitPane from 'react-split-pane';
```
Comment (targos) [inline src/component/NMRium.tsx:19]: you can remove this dependency. it's not used anymore :)

File: src/component/NMRium.tsx [hunk 2]
```diff
@@ -227,8 +219,8 @@ function NMRium({
       toggle(false);
     },
   });
-  const [isRightPanelHide, hideRightPanel] = useState(!initialShowPanels);
-  const [isResizeEventStart, setResizeEventStart] = useState(false);
+
+  const [isResizeEventStart] = useState(false);
```
Comment (targos) [inline src/component/NMRium.tsx:223]: can we remove this state? it seems useless now if it cannot be changed

Comment (lpatiny) [reply src/component/NMRium.tsx:223]: I don't remember in which case we are hiding on load the panel but it is something we could need in the future.

Not a must have before merging this PR

**Pull request merged**